iree/compiler/Conversion/LinalgToSPIRV/LinalgTileAndFusePass.cpp

```diff
@@ -43,16 +44,15 @@ namespace iree_compiler {
 // Utility functions
 //===----------------------------------------------------------------------===//
 
-static ArrayRef<int64_t> dropTrailingOnes(ArrayRef<int64_t> vector) {
-  if (vector.empty()) return vector;
-  auto numTrailingOnes = 0;
-  for (unsigned i = vector.size() - 1; i > 0; --i) {
-    if (vector[i] != 1) {
-      break;
-    }
-    numTrailingOnes++;
+/// Returns the workgroup size currently set on the function that the current
```

Comment:
According to the comment, if we expect the argument being a function, we should use funcOp here?